packages/features/ee/teams/pages/team-profile-view.tsx

```diff
@@ -84,7 +84,7 @@ const ProfileView = () => {
   });
 
   const { data: team, isLoading } = trpc.viewer.teams.get.useQuery(
-    { teamId },
+    { teamId, includeTeamLogo: true },
```

Comment:
We need the logo for team profile, so making it possible to retrieve it under a flag to have better control